comeet/UI/MyAgenda/MyAgendaViewController.swift

```diff
@@ -27,23 +27,22 @@ class MyAgendaViewController: BaseViewController {
     
     override func viewWillAppear(_ animated: Bool) {
         if let vm = viewModel {
-            if vm.showGuide  {
-                let when = DispatchTime.now() + 3 // change 2 to desired number of seconds
-                DispatchQueue.main.asyncAfter(deadline: when) { [weak self] () in
-                    // Your code with delay
-                    if let s = self {
-                        if ( s.guideView.isHidden == false) {
-                            s.guideView.isHidden = true
-                        }
-                        s.viewModel?.showGuide = false
-                    }
-                }
-            } else {
+            if vm.showGuide {
+                self.guideView.isHidden = false;
+            }
+            else {
                 self.guideView.isHidden = true;
             }
         }
     }
     
+    func hideGuideView() {
+        if ( self.guideView.isHidden == false) {
+            self.guideView.isHidden = true
+        }
+        self.viewModel?.showGuide = true
```

Comment:
If we make it false here then we need to change the code in viewModel to invert the logic. There is a bug in ViewModel where if showGuide is set to false it shows the guide.